skyflow/service_account/_validity.py

```diff
@@ -12,6 +12,9 @@ def is_expired(token: str):
     '''
     Check if stored token is not expired, if not return a new token, 
     if the token has expiry time before 5min of current time, call returns False
```

Comment:
Checks if a token is expired. If the token is expired or will expire within 5 minutes, returns `True`. Otherwise, returns `False`.